### PR TITLE
Localize Timestamps on Assesment Submissions Table

### PR DESCRIPTION
**Pull request by Hamms**
Also wrap 'em in a time element rather than just rendering them as string.

language code | screenshot
--- | ---
en-US | image
es-MX | image
ar-SA | image
de-DE | image

Also, we were rendering multiple elements on this page with the id `timestampCell`; HTML IDs should be unique per page. I updated those elements to use a class, instead.

## Links

Jira: https://codedotorg.atlassian.net/browse/PP-116

## Testing story

Tested locally in English, Spanish, German, and Arabic. Also added some unit tests.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jamescodeorg — COMMENTED** [review]
Could you paste a screenshot of what this looks like in a locale that looks different than English?

File: apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx [hunk 1]
```diff
@@ -85,21 +86,34 @@ class SubmissionStatusAssessmentsTable extends Component {
...
         break;
       default:
-        submissionTimeStampText = i18n.notStarted();
+        submissionTimeStampContent = i18n.notStarted();
     }
 
     return (
-      <div style={styles.main} id="timestampCell">
-        <div style={styles.text}>{submissionTimeStampText}</div>
+      <div style={styles.main} className="timestampCell">
```
Comment (jamescodeorg) [inline apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx:115]: For my education, why is `className` better than `id` here?

**Review by jamescodeorg — APPROVED** [review]
Nice improvement!

File: apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx (hunk 1 above)
Comment (Hamms) [reply apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx:115]: HTML IDs should be globally unique; see https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id and https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors/Type_Class_and_ID_Selectors#id_selectors

Comment (jamescodeorg) [reply apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx:115]: Oh, I see, thanks for the explanation!

**Pull request merged**